metaflow/mflog/__init__.py

```diff
@@ -41,11 +41,11 @@
 #   - python formatting: double bracket to prevent unwanted formatting
 #   - quotation marks escape adjustment for proper concatanation
 BASH_MFLOG_KFP =\
-    'mflog(){{ '\
+    'mflog(){ '\
         'T=$(date -u -Ins|tr , .); '\
-        'echo \"[MFLOG|0|${{T:0:26}}Z|%s|$T]$1\"'\
+        'echo \"[MFLOG|0|${T:0:26}Z|%s|$T]$1\"'\
             ' >> $MFLOG_STDOUT; echo $1; '\
-     ' }}' % TASK_LOG_SOURCE
+     ' }' % TASK_LOG_SOURCE
```

Comment:
why the change here? can you please undo?

Reply:
The reason for this change is that previously with `python_op`, Python implicitly removed the outer brackets (`{`, `}`). Since we're passing now as a bash command, we need to explicitly remove the extra brackets or bash throws a syntax error.

Reply:
Is this any different now from `BASH_MFLOG 
 https://github.com/zillow/metaflow/blob/9ada443dff462c940aea5da1335767ef7796746f/metaflow/mflog/__init__.py#L33-L38?`

Reply:
A very small difference;), see here: https://www.diffchecker.com/9T6O8Pg2

Reply:
Happy to link the above diff^ in the PR if you think that would add clarity.

Reply:
Does the `BASH_MFLOG` (5 lines above) work for you now? I had to create a different string given the python formatting but since you change the implementation maybe `BASH_MFLOG_KFP` is not needed.

Reply:
It technically works, but throws a number of warnings in the logs:
```
environment: 0: command not found
environment: 2021-11-12T18:58:43.873100Z: command not found
environment: task: command not found
environment: 2021-11-12T18:58:43.873100031+00:00]Setting: command not found
```
which is due to the quotation escaping. So I'll keep `BASH_MFLOG_KFP` for now.